### DISCUSSION TITLE
Fiscal year

### DISCUSSION THREAD

**Discussion by AndriyZ** [Hacker0x01/react-datepicker#4074] [Q&A]
Hi, community.
Is there a way to change the default year-month view?
Now the year starts in Jan. Is there any way to set another month as the year starts?

For example like this:
![image](https://github.com/Hacker0x01/react-datepicker/assets/13786382/513b7da5-1ca2-4792-9390-d16961b521f7)

**Comment by mininav** [comment]
What is the purpose of doing this?
